### ISSUE TITLE
jupyter notebook run indicator in upper right -- maybe delay it a little to keep it from *flickering*

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#6568]
In JupyterLab RTC testing, tonyfast noticed (and so did I!) that the kernel run indicator does a lot of disturbing flickering when other people are doing work.   CoCalc also does this to some extent.   I suggested

> "The kernel blinking might be less disturbing if the indicator *at the top* isn't shown unless the kernel has been running for at least 0.5s.  Then the flash is much less likely to appear.    There should be an immediate local indication in the cell itself that the person running the cell sees."

I like that idea, and should consider that for cocalc.
